docker-compose.yml

```diff
@@ -41,6 +41,12 @@ services:
       - DATABASE_URL=postgresql://postgres:postgres@db:5432/zeva?schema=public
       - DATABASE_URL_OLD=postgresql://postgres:postgres@host.docker.internal:5432/zeva?schema=public
       - CHECKPOINT_DISABLE=1
+      - CHES_AUTH_URL=https://dev.loginproxy.gov.bc.ca/auth/realms/comsvcauth/protocol/openid-connect/token
```

Comment:
Can we just have the definitions of these variables but not assign them any values? I know these are just dev values, but I think it would be better not to have the values in a public repo; you can have a look at https://github.com/bcgov/itvr/blob/release-1.31.0/docker-compose.yml#L60 as an example. Thanks!